README.md

```diff
@@ -1,59 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+## My portfolio mobile version
+
+This project is a mobile version of my portfolio. It will be continous project that will show what I am capable of doing a developer.
+
+Project day 3 of Microserve bootcamp, module 1
 
-> One paragraph statement about the project.
 
 ![screenshot](./app_screenshot.png)
```

Comment:
- [ ] Please fix the screenshot of the project in order to keep your repo professional.
_Please keep in mind that such a small mistake might leave a bad impression on future viewers._